styleguide.config.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
     tocMode: 'collapse',
     pagePerSection: true,
     skipComponentsWithoutExample: true,
+    styleguideComponents: {
+        Wrapper: path.resolve(__dirname, 'src/lib/containers/RenderIfInView'),
```

Comment:
This wraps every component in the styleguidist component demo page with this component (which is really the only way to go since the All Components section is only going to get bigger!)